cmd/podman/login.go

```diff
@@ -48,7 +51,8 @@ func init() {
 	completion.CompleteCommandFlags(loginCommand, auth.GetLoginFlagsCompletions())
 
 	// Podman flags.
-	flags.BoolVarP(&loginOptions.tlsVerify, "tls-verify", "", false, "Require HTTPS and verify certificates when contacting registries")
+	flags.BoolVar(&loginOptions.tlsVerify, "tls-verify", false, "Require HTTPS and verify certificates when contacting registries")
+	flags.String("secret", "", "retrieve passwd from secret file")
```

Comment:
missing autocompletion function, this should complete secrets `AutocompleteSecrets()`